### ISSUE TITLE
Cache is always missed

### ISSUE THREAD

**Issue by valendesigns** [pressjitsu/pj-page-cache-red#20]
```
curl -v http://tide.local/ -o /dev/null
*   Trying 127.0.0.1...
* TCP_NODELAY set
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0* Connected to tide.local (127.0.0.1) port 80 (#0)
> GET / HTTP/1.1
> Host: tide.local
> User-Agent: curl/7.50.3
> Accept: */*
> 
< HTTP/1.1 200 OK
< Server: nginx
< Date: Fri, 20 Apr 2018 06:37:48 GMT
< Content-Type: text/html; charset=UTF-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Vary: Accept-Encoding
< X-Powered-By: PHP/7.2.4
< X-Pj-Cache-Status: miss
< 
{ [16148 bytes data]
100 16142    0 16142    0     0   3131      0 --:--:--  0:00:05 --:--:--  104k* Curl_http_done: called premature == 0
100 54281    0 54281    0     0  10529      0 --:--:--  0:00:05 --:--:--  351k
* Connection #0 to host tide.local left intact
```

Any ideas on what to try based on the message I keep getting back from the `curl` request?

**Comment by soulseekah** [comment]
Is WP_DEBUG enabled? Is Redis running? Anything in the debug log?

**Comment by valendesigns** [comment]
Thanks for the quick reply. Yes `WP_DEBUG` is/was enabled. I disabled it and now able to get a hit. Is there any way to tell on the front-end that the page is served from cache?

**Comment by soulseekah** [comment]
That's weird, `WP_DEBUG` should not affect cache at all. Use your browser's inspector to look at the headers on the frontend. Hope this helps.

**Comment by valendesigns** [comment]
There's an undefined variable `deleted` on line 127 that is causing it.

**Comment by soulseekah** [comment]
Causing what?

Are you using our latest master branch? https://github.com/pressjitsu/pj-page-cache-red/blob/master/advanced-cache.php

**Comment by valendesigns** [comment]
No because I'm using composer `"wpackagist-plugin/pj-page-cache-red": "0.8.2"` and you haven't deployed 0.8.3 to wp.org yet. I could pull `dev-master` but that's not really great in production when it's deployed to App Engine since a change should be tested locally first.

**Comment by valendesigns** [comment]
Can't do `dev-master` either, since that's also pointing to wp.org. I'd have to add it via git and a custom repo instead. Do you have plans to push the latest version up to wp.org?

**Comment by soulseekah** [comment]
Done.

**Comment by valendesigns** [comment]
@soulseekah You need to update the `readme.txt` both the tested up to value and the version.

**Comment by soulseekah** [comment]
Pretty sure we did, it's just cached or something. Give it some time?

**Comment by valendesigns** [comment]
https://github.com/pressjitsu/pj-page-cache-red/blob/master/readme.txt#L4-L6

**Comment by valendesigns** [comment]
Also what you last pushed to wp.org got a bit messed up. This is the directory structure when you download it now.

<img width="233" alt="dirs" src="https://user-images.githubusercontent.com/62798/39042471-c50ec120-443f-11e8-9131-79c272acfd2e.png">

**Comment by soulseekah** [comment]
Not sure what you're using to pull, but:

![image](https://user-images.githubusercontent.com/685880/39043813-d1fb6308-44a7-11e8-8465-019dd241eb71.png)

The structure is absolutely fine.

**Comment by soulseekah** [comment]
Oh, in 0.8.3 crap. Yeh, my bad. I hate SVN so much :)

**Comment by valendesigns** [comment]
Still have to update the readme.txt in master for the version to bump once you push it to .org though.

**Comment by soulseekah** [comment]
GitHub master has nothing to do with .org

https://wordpress.org/plugins/pj-page-cache-red/

**Comment by valendesigns** [comment]
The readme.txt is what tells the WP plugins API what version your plugin is. So if you're manually updating the readme.txt in svn you're now missing those changes and svn isn't in sync with your version on github.

**Comment by soulseekah** [comment]
That's fine. As long as it's working for you right now. The rest can wait.

**Comment by valendesigns** [comment]
Well, it's not working because `trunk`, which has version 0.8.3, is still in the tags root and the files for `0.8.2` are still in there.

**Comment by valendesigns** [comment]
Testing locally by checking out the new version does appear to work correctly. Once I get it setup on App Engine and connected to a Redis cluster I'll let you know if there are any other issues. Thanks for working on this so quickly by the way. Looking forward to removing the github hack in my composer.json. 😄

**Comment by valendesigns** [comment]
wpackagist is now working correctly and the directory structure is fixed for 0.8.3. Thank you!
